src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/tests/overlays/ExternalLinkTypeOverlay.test.js

```diff
@@ -84,6 +84,27 @@ test('Pass correct props to Dialog', () => {
     expect(externalLinkOverlay.find('Dialog').prop('open')).toEqual(false);
 });
 
+test('Display given URL with query parameters in href input', () => {
+    const targetChangeSpy = jest.fn();
+    const urlChangeSpy = jest.fn();
+
+    const externalLinkOverlay = shallow(
+        <ExternalLinkTypeOverlay
+            href="http://www.sulu.io?param=value-1"
```

Comment:
```suggestion
            href="http://www.sulu.io/contact-us?param=value-1"
```

use a valid url. Domain without `/` is not a valid url it always requires a path part and for homepage it is `/`.